tests/Transports.Subscriptions.WebSockets.Tests/WebSocketWriterPipelineTests.cs

```diff
@@ -39,7 +39,7 @@ public WebSocketWriterPipelineFacts()
                             }
                         }
                     },
-                    83
+                    95
```

Comment:
Since `type` is always serialized (a required field), the string `"type":null,` is added to most of these tests that didn't supply a type to the serialized data.  `"type":null,` is 12 characters, so I needed to add 12 here.

Reply:
"Required" but actually is null + 
![изображение](https://user-images.githubusercontent.com/21261007/152691002-fb6db6e1-07f2-4bba-8573-728badb52810.png)
If I remember correctly I already suggested to serialize `type` only if it is non null.

Reply:
Yes, we already discussed this.

Reply:
If you want to change the behavior to skip `type`, just write a PR to do so.  Ok with me.

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/pull/2911